rest/student-rest.js

```diff
@@ -0,0 +1,22 @@
+const { DataTypes } = require("sequelize");
+const db = require(__dirname + "/../database");
+const studentCrud = require(__dirname + "/../crud/student-crud.js");
+const student = new studentCrud(db, DataTypes);
```

Comment:
No need for this. We just import the necessary function from the crud js file

Reply:
Done